app/main/util/render_utils.py

```diff
@@ -84,6 +84,16 @@ def generate_pdf_manifest(record_id):
         "main.download_record", record_id=record_id, _external=True, render=True
     )
 
+    presigned_url = None
+    try:
+        presigned_url = create_presigned_url(file)
```

Comment:
I searched on main and noticed there's no test for `create_presigned_url`, could we add one for it too

Reply:
Sure will add